apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -143,59 +132,51 @@ type SelectAppsFormValues = {
+    }
+    isValid: boolean
+}
+
+export type TScope = {
+    selectedUnitNameKeys: string[] | null
+    selectedPropertiesId: string[] | null
+    isAllPropertiesChecked: boolean | null
+    selectedSectionKeys: string[] | null
+}
+
+export const HiddenBlock = styled.div<{ hide?: boolean }>`
   ${({ hide }) => hide ? 'display: none;' : ''}
 `
 
 //TODO(DOMA-6846) wrap form label with 0 margin and use default spacing (details in 6613 pr)
-const NO_RESIZE_STYLE: React.CSSProperties = { resize: 'none' }
-const FLEX_START_STYLE: React.CSSProperties = { alignItems: 'flex-start' }
-const BIG_MARGIN_BOTTOM_STYLE: React.CSSProperties = { marginBottom: '60px' }
-const MARGIN_BOTTOM_32_STYLE: React.CSSProperties = { marginBottom: '32px' }
-const MARGIN_BOTTOM_38_STYLE: React.CSSProperties = { marginBottom: '38px' }
-const MARGIN_BOTTOM_10_STYLE: React.CSSProperties = { marginBottom: '10px' }
-const MARGIN_BOTTOM_24_STYLE: React.CSSProperties = { marginBottom: '24px' }
-const MARGIN_TOP_8_STYLE: React.CSSProperties = { marginTop: '8px' }
-const MARGIN_TOP_44_STYLE: React.CSSProperties = { marginTop: '44px' }
-const FORM_FILED_COL_PROPS = { style: { width: '100%', padding: 0, height: '44px' } }
+export const NO_RESIZE_STYLE: React.CSSProperties = { resize: 'none' }
+export const FLEX_START_STYLE: React.CSSProperties = { alignItems: 'flex-start' }
+export const BIG_MARGIN_BOTTOM_STYLE: React.CSSProperties = { marginBottom: '60px' }
+export const MARGIN_BOTTOM_32_STYLE: React.CSSProperties = { marginBottom: '32px' }
+export const MARGIN_BOTTOM_38_STYLE: React.CSSProperties = { marginBottom: '38px' }
+export const MARGIN_BOTTOM_10_STYLE: React.CSSProperties = { marginBottom: '10px' }
+export const MARGIN_BOTTOM_24_STYLE: React.CSSProperties = { marginBottom: '24px' }
+export const MARGIN_TOP_8_STYLE: React.CSSProperties = { marginTop: '8px' }
+export const MARGIN_TOP_44_STYLE: React.CSSProperties = { marginTop: '44px' }
+export const FORM_FILED_COL_PROPS = { style: { width: '100%', padding: 0, height: '44px' } }
 export const SCROLL_TO_FIRST_ERROR_CONFIG: ScrollOptions = { behavior: 'smooth', block: 'center' }
 export const SHOW_TIME_CONFIG = { defaultValue: dayjs('00:00:00:000', 'HH:mm:ss:SSS') }
 export const FULL_WIDTH_STYLE: React.CSSProperties = { width: '100%' }
-const SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 24]
-const EXTRA_SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 10]
-const BIG_HORIZONTAL_GUTTER: [Gutter, Gutter] = [50, 0]
-const ALL_SQUARE_BRACKETS_OCCURRENCES_REGEX = /\[[^\]]*?\]/g
-const ADDITIONAL_DISABLED_MINUTES_COUNT = 5
-const DATE_FORMAT = 'DD MMMM YYYY HH:mm'
+export const SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 24]
+export const EXTRA_SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 10]
+export const BIG_HORIZONTAL_GUTTER: [Gutter, Gutter] = [50, 0]
+export const ALL_SQUARE_BRACKETS_OCCURRENCES_REGEX = /\[[^\]]*?\]/g
+export const ADDITIONAL_DISABLED_MINUTES_COUNT = 5
+export const DATE_FORMAT = 'DD MMMM YYYY HH:mm'
```

Comment:
I do not really like exporting this stuff elsewhere, 

It is considered a bad practice in this project, better to stick to UI kit where possible, and just inline required values

You are exporting these values, this means that other engineers will use them, and project might become more coupled (https://www.geeksforgeeks.org/software-engineering-coupling-and-cohesion/) + this promotes bad practice

Lets remove these exports